### PR TITLE
A very simple interactive chat

### PR DESCRIPTION
**Pull request by duralakun**
I couldn't find an example of an interactive chat, so I tried to make a very simple one.

### CODE REVIEW

**Comment by duralakun** [comment]
If you don't like it, pls tell me why.

File: examples/chat.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
+ * This is a very simple chat client.
+ *
+ * That shows you how you can read and interact with any chat on a server
+ *
+ * Just type into the console and press enter to say something
+ *
+ * Be sure to stand somewhere save so you dont get killed by zombies :)
```
Comment (rom1504) [inline examples/chat.js]: safe

File: examples/chat.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  host: process.argv[2],
+  port: parseInt(process.argv[3]),
+  username: process.argv[4] ? process.argv[4] : 'echo',
+  password: process.argv[5],
+  verbose: true
+})
+
+// uses on 'message' instead of on 'chat'
+// because on 'chat' does not work on all servers 
+bot.on('message', (message) => {
```
Comment (rom1504) [inline examples/chat.js]: what about the fact it's json ?

Comment (plexigras) [inline examples/chat.js]: `console.log(message.toString())` might be better

**Comment by duralakun** [comment]
It says "All checks have failed" but I think this is a bug? What am I doing wrong?

**Comment by duralakun** [comment]
Thanks for the reviews I did some changes.
I was thinking about adding the on 'health' event and print out a message in the conole, to warn players before they die.
Also a quit option would be nice. So you can type for example ".quit" in the console and it will call the bot.quit() function.
But this would me make the program less simple. Maybe for a more advanced chat example?

**Pull request closed without merging**